book/plugins.md

```diff
@@ -28,6 +28,18 @@ Core plugins are typically distributed with the Nushell release and should alrea
 
 ::: tip Installing using Cargo
 For example, when installing or upgrading Nushell directly from crates.io using `cargo install nu`, the corresponding core plugins for that version may also be installed or updated using `cargo install nu_plugin_<plugin_name>`.
+
+To install all of the default plugins, from within Nushell run:
+
+```nu
+[ nu_plugin_inc
+  nu_plugin_polars
+  nu_plugin_gstat
+  nu_plugin_formats
+  nu_plugin_query
+] | each { cargo install $in; $in }
```

Comment:
what's the point of running $in. shouldn't we be doing `plugin add $in`?

Reply:
Just to display/return the results, rather than an empty list.  Could do an `ignore` or `for` instead; it just seemed like this was cleaner.

Reply:
If you want to give feedback then I think print would be better. I think running the plugin gives the wrong idea because we really don't recommend running them as nu_plugin_name except for diagnostic purposes.

Reply:
> we really don't recommend running them as nu_plugin_name except for diagnostic purposes.

`$in` doesn't run the plugin ;-).  It simply populates a list with the *name* of the plugin that was passed in.

Take out the `cargo install` statement, and it's like saying:

```nu
[ 1 2 3 ] | each { $in }
```

Are you okay with it in that case?

I have to retract what I said above, too - a `print` won't solve it, since it would still generate the empty list as well.  Here's what it would look like:

```nu
[ nu_plugin_inc
∙   nu_plugin_polars
∙   nu_plugin_gstat
∙   nu_plugin_formats
∙   nu_plugin_query
∙ ] | each { print $'Installing ($in)'; cargo install $in }
Installing nu_plugin_inc
    Updating crates.io index
     Ignored package `nu_plugin_inc v0.98.0` is already installed, use --force to override
Installing nu_plugin_polars
    Updating crates.io index
     Ignored package `nu_plugin_polars v0.98.0` is already installed, use --force to override
Installing nu_plugin_gstat
    Updating crates.io index
     Ignored package `nu_plugin_gstat v0.98.0` is already installed, use --force to override
Installing nu_plugin_formats
    Updating crates.io index
     Ignored package `nu_plugin_formats v0.98.0` is already installed, use --force to override
╭───┬──╮
│ 0 │  │
│ 1 │  │
│ 2 │  │
│ 3 │  │
╰───┴──╯
Installing nu_plugin_query
    Updating crates.io index
     Ignored package `nu_plugin_query v0.98.0` is already installed, use --force to override
╭───┬──╮
│ 4 │  │
╰───┴──╯
```

And a `for` can't be cleanly (IMHO) done as a one-liner.

I'll update to `ignore`, I guess.